COPYRIGHT

```diff
@@ -80,3 +80,23 @@ THAT OPERATION WILL BE UNINTERRUPTED OR ERROR FREE.  The Regents of the
 University of Michigan and Merit Network, Inc. shall not be liable for any
 special, indirect, incidental or consequential damages with respect to any
 claim by Licensee or any third party arising from use of the software.
+------------------------------------------------------------------------------
```

Comment:
Please no new license. Just add the Copyright at the beginning of the file.

Reply:
Just to clarify: After line 8 in this file, right? Will do.